dynamic_dynamodb/__init__.py

```diff
@@ -101,6 +104,126 @@ def main():
     except Exception as error:
         logger.exception(error)
 
+def create_time_series_tables(rotate_suffix, rotate_window_start, rotate_scavenge, table_name, table_key, tables_and_gsis, next_table_names):
```

Comment:
Can you combine these two methods into one? I think most of the logic is the same except getting the next/prev table name of a period